src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -15,7 +15,7 @@ module GEOS_LandAssimGridCompMod
   use ESMF
```

Comment:
The comments ("DESCRIPTION") before this line should be edited because LandAssimGC now does have Children.